openedx/core/djangoapps/xblock/rest_api/views.py

```diff
@@ -258,19 +255,10 @@ def post(self, request, usage_key_str):
         # Save after the callback so any changes made in the callback will get persisted.
         block.save()
 
-        # Raise a signal to indicate the library block was updated
-        try:
-            library_block_key = LibraryUsageLocatorV2.from_string(usage_key_str)
-            LIBRARY_BLOCK_UPDATED.send_event(
-                library_block=LibraryBlockData(
-                    library_key=library_block_key.context_key,
-                    usage_key=library_block_key,
-                )
-            )
-
-        except InvalidKeyError:
-            # TODO raise XBLOCK_UPDATED if this API is ever used for course blocks.
-            pass
+        # Signal that we've modified this block
+        usage_key = UsageKey.from_string(usage_key_str)
```

Comment:
I think usage_key is already defined earlier in this function scope, so this line isn't needed.

Reply:
Oops! Sorry.. fixed with [ceaf314](https://github.com/openedx/edx-platform/pull/35103/commits/ceaf3142d5599796b87505ac418d69283fd9fe68)